collections/_priorities/rendering-08-overhaul-lightmapgi.md

```diff
@@ -0,0 +1,16 @@
+---
+last_updated: "2024-11-30"
+category: rendering
+rank: 8
+created: "2024-11-30"
+stage:
+  # Must be dates (YYYY-MM-DD), empty if not there yet.
+  planning: ""
+  ongoing: ""
+  done: ""
+anchor: "overhaul-lightmapgi"
+title: "Overhaul `LightmapGI` to improve baking workflow, performance, and features"
+description: |
+  `LightmapGI` needs a lot of polish and improvements for us to meet the goals we have set for it. Baking times are slower than we want and it often takes too much manual effort to get bakes to achieve the quality that users need.
```

Comment:
Perhaps link to this tracker issue? https://github.com/godotengine/godot/issues/56033